app/templates/macros/toolkit_forms.html

```diff
@@ -209,3 +209,38 @@
     </fieldset>
   {% endif %}
 {%- endmacro %}
+
+{% macro multiquestion(question_content, service_data, errors, question_number=None, get_question=None) -%}
+<div class="question" id="{{ question_content.id }}">
+  <label for="input-{{ question_content.id }}">
+      <span class="question-heading{% if question_content.hint %}-with-hint{% endif %}">
+          <span class="question-number">
+            {{ question_number }}
+          </span>
+        
+        {{ question_content.question }}
+        
+      </span>
+
+  </label>
+  <legend>
+    <span class="visually-hidden">{{ question_content.name }}</span>
+  </legend>
```

Comment:
[`<legend>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend) are meant to be used within a `<fieldset>` element, and this looks like a pretty good candidate for being a fieldset.

The other thing I've just noticed is that, according to the MDN guide:

> `<legend>`
> Permitted Parents: A `<fieldset>` whose first child is this `<legend>` element

This isn't how [the `boolean_list` template](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/macros/toolkit_forms.html#L174) is structured.

If it's an easy change to make, do you mind swapping the `<span class="question-heading">` and the `<legend>` elements in the above template as well? (If not, I can do it.)